modules/test/playwright/helpers/ApiHelpers.ts

```diff
@@ -3,25 +3,32 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
+import {Page} from '@playwright/test';
+
 import {liferayConfig} from '../liferay.config';
 import {FeatureFlagApiHelper} from './FeatureFlagApiHelper';
 import {ObjectAdminApiHelper} from './ObjectAdminApiHelper';
 
 export class ApiHelpers {
-	constructor(page) {
+	readonly baseUrl: string;
+	readonly featureFlag: FeatureFlagApiHelper;
+	readonly objectAdmin: ObjectAdminApiHelper;
+	readonly page: Page;
+
+	constructor(page: Page) {
 		this.baseUrl = liferayConfig.environment.baseUrl + '/o/';
 		this.featureFlag = new FeatureFlagApiHelper(page);
 		this.objectAdmin = new ObjectAdminApiHelper(this);
 		this.page = page;
 	}
 
-	async delete(url) {
+	async delete(url: string) {
 		return this.page.request.delete(url, {
 			headers: await this.getHeader(),
 		});
 	}
 
-	async post(url, data) {
+	async post(url: string, data: object) {
```

Comment:
`object` type could accept a function, like `() => {}`, so [I have created](https://github.com/liferay-objects/liferay-portal/pull/2996/commits/d8cdada113d097be18f94bef7d16d399facd39ea) a new interface for generic data objects.